src/lib/core/utils/data-element-constraints.ts

```diff
@@ -170,6 +178,12 @@ export function mergeConstraints(
     keys.map((key): [string, DataElementConstraint] => {
       const constraintA = constraintMapA[key];
       const constraintB = constraintMapB[key];
+      const mergedIsTemporal =
+        constraintA == null || constraintB == null
+          ? undefined
+          : constraintA.isTemporal && constraintB.isTemporal
+          ? true
+          : undefined;
```

Comment:
This doesn't seem quite right, to me. If `isTemporal` is `false` for both constraints, or if only one has it set, then the merged value is `undefined`. I think if only one is set, then that one should be the merged value. If both are set, then the `&&` evaluation should be the merged value:

```typescript

      const mergedIsTemporal =
        constraintA == null || constraintB == null
          ? undefined
          : constraintA.isTemporal == null ? constraintB.isTemporal
          : constraintB.isTemporal == null ? constraintA.isTemporal
          : constraintA.isTemporal && constraintB.isTemporal;
```

I suppose this also depends on how we treat the absence of `isTemporal`...

Reply:
i think youre right about the case where both are false, the merged result should also be false. but if either is undefined, i take that to mean its not restricted on temporal terms either way, and the merged result shouldnt be either.